### PR TITLE
Enable SigMF plugins on windows

### PR DESCRIPTION
**Pull request by srcejon**
This patch enables SigMF plugins on windows.

This pull request: https://github.com/f4exb/sdrangel-windows-libraries/pull/15 for sdrangel-windows-libraries needs to be applied first, as it contains the DLL built for windows.

And just in case you didn't see it, this pull request: https://github.com/f4exb/libsigmf/pull/2 contains the fix for libsigmf that allows it to be built using MSVC.

### CODE REVIEW

File: sdrbase/CMakeLists.txt [hunk 1]
```diff
@@ -519,6 +518,10 @@ target_link_libraries(sdrbase
     qrtplib
     swagger
 )
+if (LIBSIGMF_FOUND)
+    target_link_libraries(sdrbase optimized ${LIBSIGMF_LIBRARIES})
+    target_link_libraries(sdrbase debug ${LIBSIGMF_LIBRARIES_DEBUG})
```
Comment (f4exb) [inline sdrbase/CMakeLists.txt:523]: Cmake fails here:
```
#78 5.141 CMake Error at sdrbase/CMakeLists.txt:523 (target_link_libraries):
#78 5.141   The "debug" argument must be followed by a library.
```

**Comment by srcejon** [comment]
To fix the Windows build, I think you need to update the external/windows link to point to the latest version of sdrangel-windows-libraries.

Hopefully the above patch should fix the linux build.

**Comment by f4exb** [comment]
Now drangel-windows-libraries is the latest on master. Next to fix Windows you need to do a `git submodule update` in your fork/branch

**Comment by f4exb** [comment]
Done on master

**Comment by srcejon** [comment]
Thanks!

**Pull request merged**